app/src/main/kotlin/com/wire/android/feature/auth/login/LoginViewModel.kt

```diff
@@ -39,14 +48,17 @@ class LoginViewModel @Inject constructor(
     }
 
     fun onUserIdentifierChange(newText: String) {
-        userIdentifier.value = newText
-        savedStateHandle.set(USER_IDENTIFIER_SAVED_STATE_KEY, userIdentifier.value)
+        userIdentifier = newText
+        savedStateHandle.set(USER_IDENTIFIER_SAVED_STATE_KEY, userIdentifier)
     }
 
     fun onPasswordChange(newText: String) {
-        password.value = newText
+        password = newText
     }
 
+    suspend fun navigateToConvScreen() =
+        navigationManager.navigate(NavigationCommand(NavigationItem.Home.navigationRoute("home_conversations"), true))
+
     private companion object {
         const val USER_IDENTIFIER_SAVED_STATE_KEY = "user_identifier"
     }
```

Comment:
it looks like this ViewModel uses a MVVM approach, it is recommended by Google to use a MVI approach so I would suggest to refactor it, maybe it is something we need to discuss internally first ?

Reply:
this need more discussien with the team, for new i will leave the viewModel as it is